### PR TITLE
TEAMROCKET#75: User Personalization

### PR DESCRIPTION
**Pull request by gmihir**
A user can enter their preferences (automatic prompt on sign up). Their preferences affect the recommended recipes they see as well as the ordering of their search results.

### CODE REVIEW

**Comment by gmihir** [comment]
> Just to clarify - this is not yet connected to the database right? So it's expected that my survey does not persist after I log out?
> 
> Also, if I don't fill out the survey and just submit it then there are no recommended recipes when I get redirected to the home page. I think my expectation was that I would just be recommended recipes without any filtering (rather than be given no recommendations). Not sure if the intention was to just not recommend recipes, though.

should be fixed

File: root/html/survey.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                </fieldset>
+        
+                <h3> What are your goals in cooking?</h3>
+                <fieldset name="goals">
+                    <input value="fiveIngredientsOrLess" id="fiveIngredientsOrLess" type="checkbox" name="surveyPreference" placeholder="Easy Recipes" />
+                    <label for="easyRecipes">I'm looking for recipes that are easy to make.</label><br>
+                    <!-- <input id="cheapRecipes" type="checkbox" class="surveyGoal" placeholder="Cheap Recipes" />
+                    <label for="cheapRecipes">Cheap Recipes</label><br> -->
+                    <input value="quickEat" id="quickEat" type="checkbox" name="surveyPreference" placeholder="Quick Recipes" />
+                    <label for="quickRecipes">I'm often in a hurry so I want quick recipes.</label><br>
```
Comment (lramos0) [inline root/html/survey.html:43]: Nice! Wish we could have more cooking goals than these two, but understandable for the sake of time 😞

**Review by lramos0 — APPROVED** [review]
Looks Awesome, glad we got to account for some user preferences/personalizations!

File: root/html/survey.html (hunk 1 above)
Comment (gmihir) [reply root/html/survey.html:43]: It's pretty easy to add more! If you can think of a preference that involves using the tags we have for recipes (vegetarian, vegan, glutenFree, dairyFree, quickEat, fiveIngredientsOrLess, breakfast, lunch, dinner, readyInMinutes, servings), I can add them

**Pull request merged**